frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,57 @@
+import { Column, useTable } from 'react-table';
+import { FC } from 'common/types/types';
+import styles from './table/styles.module.scss';
+
+type Props<Data extends Record<string, unknown>> = {
+  columns: Column<Data>[];
+  data: readonly Data[];
+};
+const Table: FC<Props<Record<string, unknown>>> = ({ columns, data }) => {
```

Comment:
We should pass data type from outside
```suggestion
const Table = <Data extends Record<string, unknown>>({ columns, data }: Props<Data>): ReactElement => {
```